statchatui/src/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import "./SignUp.css";
+import { Link } from "react-router-dom";
+
+const SignUp = () => {
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    const formData = {
+      Username: event.target.username.value,
+      Email: event.target.email.value,
+      Password: event.target.password.value,
+    };
+    try {
+      const response = await fetch("http://localhost:3002/users", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(formData),
+      });
+
+      if (response.ok) {
+        alert("Sign-up successful!");
```

Comment:
Let's come back to this and turn these alerts into UI components! Can you create a task for this? (either in your google doc, or in github issues)

Reply:
okay i will create an issue and come back to this